templates/macros.html

```diff
@@ -3,11 +3,14 @@
 		<p style="text-align: center;"><b>Bugs</b></p>
 		{% for bug in row.bugs %}
 			<ul>
-			{% if bug.bug_url != None %}
-				<li><a href="{{bug.bug_url}}">{{bug.bug_name}}</a></li>
-			{% else %}
-				<li>{{bug.bug_name}}</li>
-			{% endif %}
+				<li>
+				{% if bug.bug_url != None %}
+					<a href="{{bug.bug_url}}">{{bug.bug_name}}</a>
+				{% else %}
+					{{bug.bug_name}}
+				{% endif %}
+				{{ build_numbers(row) }}
+				</li>
```

Comment:
Please make the indentation consistent

Reply:
@nathan-weinberg What do you mean? It is consistent in this file. It is based on composes(row) indention. Now I noticed that, composes(row) is probably not consistent with other html files. Are you asking for fixing indention in the whole file?

Reply:
Looking again, this may be an artifact of how GitHub displays indentation. The raw file seems fine - you can disregard this comment.